specs/commandline/inspect.md

```diff
@@ -78,7 +78,13 @@ localhost:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac4efe37a5380ee
     │   ├── user defined attributes
     │   │   └── io.wabbit-networks.buildId: 123                             //user defined metadata
     │   ├── unsigned attributes
-    │   │   ├── io.cncf.notary.timestampSignature: <Base64(TimeStampToken)> //TSA response
+    │   │   ├── timestampSignature                                          //TSA response
+    |   │   │   ├── timestamp: [Fri Jun 23 22:04:31 2023, Fri Jun 23 22:04:31 2023]
```

Comment:
> [Fri Jun 23 22:04:31 2023, Fri Jun 23 22:04:31 2023]

The upper bound and lower bound are the same, so showing the time range may not be useful if the accuracy is less than 1 second. Therefore, can we only show one time rather than a range if the accuracy is less than 1 second?

Reply:
This output is from tspclient-go and by design. So I would say keep it as is for this version.